haskellPackages: update stackage and hackage

cdepillabout opened the pull request:
### This Merge

This PR is the regular merge of the `haskell-updates` branch into `master`.

This branch is being continually built and tested by hydra at https://hydra.nixos.org/jobset/nixpkgs/haskell-updates. You may be able to find an up-to-date Hydra build report at [cdepillabout/nix-haskell-updates-status](https://github.com/cdepillabout/nix-haskell-updates-status).

We roughly aim to merge these `haskell-updates` PRs at least once every two weeks. See the @NixOS/haskell [team calendar](https://cloud.maralorn.de/apps/calendar/p/H6migHmKX7xHoTFa) for who is currently in charge of this branch.

### haskellPackages Workflow Summary

Our workflow is currently described in [`pkgs/development/haskell-modules/HACKING.md`](https://github.com/NixOS/nixpkgs/blob/haskell-updates/pkgs/development/haskell-modules/HACKING.md).

The short version is this:
* We regularly update the Stackage and Hackage pins on `haskell-updates` (normally at the beginning of a merge window).
* The community fixes builds of Haskell packages on that branch.
* We aim at at least one merge of `haskell-updates` into `master` every two weeks.
* We only do the merge if the [`mergeable`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/mergeable) job is succeeding on hydra.
* If a [`maintained`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/maintained) package is still broken at the time of merge, we will only merge if the maintainer has been pinged 7 days in advance. (If you care about a Haskell package, become a maintainer!)

---

This is the follow-up to #194600. Come to [#haskell:nixos.org](https://matrix.to/#/#haskell:nixos.org) if you have any questions.

cdepillabout committed:
haskellPackages: stackage LTS 19.27 -> LTS 19.28

This commit has been generated by maintainers/scripts/haskell/update-stackage.sh

cdepillabout committed:
all-cabal-hashes: 2022-10-05T14:24:18Z -> 2022-10-11T19:16:50Z

This commit has been generated by maintainers/scripts/haskell/update-hackage.sh

cdepillabout committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

cdepillabout committed:
haskellPackages.colourista: remove no longer needed doJailbreak

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

markus1189 committed:
haskellPackages.http-client-websockets: unbreak

cdepillabout committed:
Merge pull request #196754 from markus1189/haskellPackages-http-client-websockets

haskellPackages.http-client-websockets: unbreak

markus1189 committed:
haskellPackages.binsm: unbreak

markus1189 committed:
haskellPackages.copilot-c99: unbreak

markus1189 committed:
haskellPackages.data-compat: unbreak

markus1189 committed:
haskellPackages.scotty-form: unbreak

markus1189 committed:
haskellPackages.uuid-bytes: unbreak

markus1189 committed:
haskellPackages.vado: unbreak

cdepillabout committed:
Merge pull request #196767 from markus1189/haskellPackages-unbreak

Unbreak haskellPackages: vado, uuid-bytes, scotty-form, data-compat, copilot-c99, binsm

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann committed:
Merge remote-tracking branch 'origin/master' into haskell-updates

sternenseemann committed:
haskellPackages.cabal2nix-unstable: 2022-10-10 -> 2022-10-22

With this hackage2nix knows that libcrypt can no longer be found in
glibc, but needs to be taken from libxcrypt. We can trim down the
overrides accordingly, leaving only the test failure workaround for
crypt-sha512.

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

msooseth committed:
smt2-parser is no longer broken

maralorn committed:
Merge pull request #197732 from msooseth/non-broken-smt2-parser

smt2-parser is no longer broken

github-actions[bot] committed:
Merge master into haskell-updates

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann commented:
### [haskell-updates build report from hydra](https://hydra.nixos.org/jobset/nixpkgs/haskell-updates)
*evaluation [1784125](https://hydra.nixos.org/eval/1784125) of nixpkgs commit [4404ed7](https://github.com/NixOS/nixpkgs/commits/4404ed785e8596b10b8fea83c13858dc2a2e198d) as of 2022-10-27 08:35 UTC*

:red_circle: **Branch not mergeable**
  * `mergeable` jobset failed.
  * `maintained` jobset failed.

#### Build summary

 | Platform | Failed :x: | DependencyFailed :heavy_exclamation_mark: | TimedOut :hourglass::no_entry_sign: | Success :heavy_check_mark: | 
 | --- | --- | --- | --- | --- | 
 | [aarch64-linux :iphone:](https://hydra.nixos.org/eval/1784125?filter=.aarch64-linux) | 31 | 30 |  | 6655 | 
 | [x86_64-darwin :apple:](https://hydra.nixos.org/eval/1784125?filter=.x86_64-darwin) | 58 | 20 | 38 | 6545 | 
 | [x86_64-linux :penguin:](https://hydra.nixos.org/eval/1784125?filter=.x86_64-linux) | 12 | 9 | 1 | 6734 | 
#### Maintained packages with build failure
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196421807) [[:apple::x:]](https://hydra.nixos.org/build/196413209) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196423566) [haskellPackages.rel8](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.rel8) @sternenseemann
#### Maintained packages with failed dependency
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196589929) [maintained](https://hydra.nixos.org/eval/1784125?filter=maintained) @cdepillabout @expipiplus1 @maralorn @sternenseemann
- [ ] [xmonad](https://hydra.nixos.org/eval/1784125?filter=xmonad) 
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196423665) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196421939) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196420008) [haskellPackages](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.xmonad)
  -   [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196589962) [nixosTests](https://hydra.nixos.org/eval/1784125?filter=nixosTests.xmonad)
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196589931) [nixosTests.xmonad-xdg-autostart](https://hydra.nixos.org/eval/1784125?filter=nixosTests.xmonad-xdg-autostart) 
#### Unmaintained packages with build failure
<details><summary>74 job(s) </summary>

- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196419181) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196410503) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196413801) [haskellPackages.OrderedBits](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.OrderedBits)  :arrow_heading_up: 5 | 36
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196404777) [[:apple::x:]](https://hydra.nixos.org/build/196416838) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196420870) [haskellPackages.persistent-postgresql](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.persistent-postgresql)  :arrow_heading_up: 5 | 23
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196410164) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196411832) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196412240) [haskellPackages.hw-json-simd](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hw-json-simd)  :arrow_heading_up: 4 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196404666) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196419446) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196404874) [haskellPackages.hw-simd](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hw-simd)  :arrow_heading_up: 4 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196424852) [[:apple::x:]](https://hydra.nixos.org/build/196404831) [[:penguin::x:]](https://hydra.nixos.org/build/196419990) [haskellPackages.country](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.country)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196416598) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196421182) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196423117) [haskellPackages.long-double](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.long-double)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196410842) [[:apple::x:]](https://hydra.nixos.org/build/196409180) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196421434) [haskellPackages.quic](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.quic)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196406329) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196406423) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196424340) [haskellPackages.freetype2](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.freetype2)  :arrow_heading_up: 1 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196409748) [[:apple::x:]](https://hydra.nixos.org/build/196406980) [[:penguin::x:]](https://hydra.nixos.org/build/196414889) [haskellPackages.union](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.union)  :arrow_heading_up: 1 | 3
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196419959) [[:apple::x:]](https://hydra.nixos.org/build/196405842) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196420157) [haskellPackages.easytensor](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.easytensor)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196407823) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196413034) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196419521) [haskellPackages.nlopt-haskell](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.nlopt-haskell)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196423680) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196422816) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196421172) [haskellPackages.swisstable](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.swisstable)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196405687) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196408197) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196422914) [haskellPackages.unicode-properties](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.unicode-properties)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196414378) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196413912) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196410936) [haskellPackages.flatparse](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.flatparse)  :arrow_heading_up: 0 | 15
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196411763) [[:apple::x:]](https://hydra.nixos.org/build/196417019) [[:penguin::x:]](https://hydra.nixos.org/build/196405427) [haskellPackages.incipit](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.incipit)  :arrow_heading_up: 0 | 11
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196405726) [[:apple::x:]](https://hydra.nixos.org/build/196420513) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196421587) [haskellPackages.PyF](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.PyF)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196404311) [[:apple::x:]](https://hydra.nixos.org/build/196404661) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196404870) [haskellPackages.hmidi](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hmidi)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196404780) [[:apple::x:]](https://hydra.nixos.org/build/196406613) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196407410) [haskellPackages.json-rpc](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.json-rpc)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196414516) [[:apple::x:]](https://hydra.nixos.org/build/196405282) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196409937) [haskellPackages.posix-socket](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.posix-socket)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196406736) [[:apple::x:]](https://hydra.nixos.org/build/196421247) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196412026) [haskellPackages.gi-gdkx11](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.gi-gdkx11)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196420329) [[:apple::x:]](https://hydra.nixos.org/build/196420237) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196415426) [haskellPackages.hamid](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hamid)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196421142) [[:apple::x:]](https://hydra.nixos.org/build/196406128) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196414969) [haskellPackages.hmatrix-morpheus](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hmatrix-morpheus)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196414104) [[:apple::x:]](https://hydra.nixos.org/build/196419330) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196405530) [haskellPackages.huckleberry](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.huckleberry)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196409957) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196410800) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196406050) [haskellPackages.picosat](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.picosat)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196422932) [[:apple::x:]](https://hydra.nixos.org/build/196417135) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196404538) [haskellPackages.select](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.select)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196422763) [[:apple::x:]](https://hydra.nixos.org/build/196404284) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196421351) [haskellPackages.sysinfo](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.sysinfo)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196405836) [[:apple::x:]](https://hydra.nixos.org/build/196404298) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196407756) [haskellPackages.FractalArt](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.FractalArt) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196405484) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196407541) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196409984) [haskellPackages.HsASA](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.HsASA) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196415896) [[:apple::x:]](https://hydra.nixos.org/build/196417770) [[:penguin::x:]](https://hydra.nixos.org/build/196413483) [haskellPackages.brick-panes](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.brick-panes) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196414727) [[:apple::x:]](https://hydra.nixos.org/build/196417614) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196422839) [haskellPackages.chiphunk](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.chiphunk) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196414618) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196407281) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196410128) [haskellPackages.comfort-fftw](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.comfort-fftw) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196415955) [[:apple::x:]](https://hydra.nixos.org/build/196422834) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196417145) [haskellPackages.epub-tools](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.epub-tools) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196406840) [[:apple::x:]](https://hydra.nixos.org/build/196412815) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196420983) [haskellPackages.fudgets](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.fudgets) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196423355) [[:apple::x:]](https://hydra.nixos.org/build/196421561) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196422326) [haskellPackages.ghc-gc-hook](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.ghc-gc-hook) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/196409032) [haskellPackages.gi-gtkosxapplication](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.gi-gtkosxapplication) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196411975) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196415100) [haskellPackages.gnome-keyring](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.gnome-keyring) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/196411014) [haskellPackages.gtk-mac-integration](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.gtk-mac-integration) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196406571) [[:apple::x:]](https://hydra.nixos.org/build/196419762) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196423902) [haskellPackages.gtk-traymanager](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.gtk-traymanager) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/196422848) [haskellPackages.gtk3-mac-integration](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.gtk3-mac-integration) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196404881) [[:apple::x:]](https://hydra.nixos.org/build/196405673) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196418904) [haskellPackages.hasql-th](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hasql-th) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196423052) [[:apple::x:]](https://hydra.nixos.org/build/196414907) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196416429) [haskellPackages.hid](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hid) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196415496) [[:apple::x:]](https://hydra.nixos.org/build/196405774) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196415190) [haskellPackages.highlight](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.highlight) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196421209) [[:apple::x:]](https://hydra.nixos.org/build/196417133) [[:penguin::x:]](https://hydra.nixos.org/build/196414810) [haskellPackages.hoauth2-providers](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hoauth2-providers) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196406900) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196411967) [[:penguin::x:]](https://hydra.nixos.org/build/196404695) [haskellPackages.hslua-list](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hslua-list) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196419952) [[:apple::x:]](https://hydra.nixos.org/build/196408502) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196404381) [haskellPackages.hsshellscript](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hsshellscript) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196409451) [[:apple::x:]](https://hydra.nixos.org/build/196424649) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196418643) [haskellPackages.hssourceinfo](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hssourceinfo) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196413166) [[:apple::x:]](https://hydra.nixos.org/build/196406507) [[:penguin::x:]](https://hydra.nixos.org/build/196413743) [haskellPackages.hxmppc](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hxmppc) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196408762) [[:apple::x:]](https://hydra.nixos.org/build/196421083) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196406479) [haskellPackages.intricacy](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.intricacy) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196417965) [[:apple::x:]](https://hydra.nixos.org/build/196409735) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196422092) [haskellPackages.ipcvar](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.ipcvar) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196420225) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196421431) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196420524) [haskellPackages.jammittools](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.jammittools) 
- [ ] [[:apple::x:]](https://hydra.nixos.org/build/196406605) [haskellPackages.kqueue](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.kqueue) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196423701) [[:apple::x:]](https://hydra.nixos.org/build/196418472) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196424451) [haskellPackages.linux-framebuffer](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.linux-framebuffer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196415646) [[:apple::x:]](https://hydra.nixos.org/build/196413167) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196414637) [haskellPackages.mediawiki2latex](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.mediawiki2latex) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196419744) [[:apple::x:]](https://hydra.nixos.org/build/196413047) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196412621) [haskellPackages.memfd](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.memfd) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196421754) [[:apple::x:]](https://hydra.nixos.org/build/196409698) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196411825) [haskellPackages.mercury-api](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.mercury-api) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196420356) [[:apple::x:]](https://hydra.nixos.org/build/196421409) [[:penguin::x:]](https://hydra.nixos.org/build/196416775) [haskellPackages.named-binary-tag](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.named-binary-tag) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196414531) [[:apple::x:]](https://hydra.nixos.org/build/196416009) [[:penguin::x:]](https://hydra.nixos.org/build/196412909) [haskellPackages.om-http](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.om-http) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196404810) [[:apple::x:]](https://hydra.nixos.org/build/196411527) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196417910) [haskellPackages.om-time](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.om-time) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196418636) [[:apple::x:]](https://hydra.nixos.org/build/196414940) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196405244) [haskellPackages.phatsort](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.phatsort) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196420631) [[:apple::x:]](https://hydra.nixos.org/build/196412790) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196413895) [haskellPackages.ping-wrapper](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.ping-wrapper) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196415295) [[:apple::x:]](https://hydra.nixos.org/build/196418054) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196415808) [haskellPackages.procex](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.procex) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196419980) [[:apple::x:]](https://hydra.nixos.org/build/196415687) [[:penguin::x:]](https://hydra.nixos.org/build/196417900) [haskellPackages.quickcheck-lockstep](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.quickcheck-lockstep) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196411469) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196420163) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196407467) [haskellPackages.risc386](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.risc386) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196419908) [[:apple::x:]](https://hydra.nixos.org/build/196413327) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196409061) [haskellPackages.sfml-audio](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.sfml-audio) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196412072) [[:apple::x:]](https://hydra.nixos.org/build/196415307) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196410900) [haskellPackages.shared-memory](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.shared-memory) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196416387) [[:apple::x:]](https://hydra.nixos.org/build/196423172) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196413707) [haskellPackages.slugify](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.slugify) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196424322) [[:apple::x:]](https://hydra.nixos.org/build/196411723) [[:penguin::x:]](https://hydra.nixos.org/build/196416287) [haskellPackages.swarm](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.swarm) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196415161) [[:apple::x:]](https://hydra.nixos.org/build/196419440) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196407591) [haskellPackages.tailfile-hinotify](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.tailfile-hinotify) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196422334) [[:apple::x:]](https://hydra.nixos.org/build/196418406) [[:penguin::x:]](https://hydra.nixos.org/build/196421460) [haskellPackages.test-lib](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.test-lib) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/196413795) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196414695) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196416632) [haskellPackages.wiringPi](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.wiringPi) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196411339) [[:apple::x:]](https://hydra.nixos.org/build/196406099) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196416612) [haskellPackages.xmonad-utils](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.xmonad-utils) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196416033) [[:apple::x:]](https://hydra.nixos.org/build/196415892) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196405200) [haskellPackages.yoga](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.yoga) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196413019) [[:apple::x:]](https://hydra.nixos.org/build/196410691) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196414757) [haskellPackages.zot](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.zot) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196423616) [[:apple::x:]](https://hydra.nixos.org/build/196419599) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196404486) [haskellPackages.zxcvbn-c](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.zxcvbn-c) 
</details>

#### Unmaintained packages with failed dependency
<details><summary>61 job(s) </summary>

- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196417357) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196416817) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196423706) [haskellPackages.esqueleto](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.esqueleto)  :arrow_heading_up: 5 | 21
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196416061) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196420657) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196404375) [haskellPackages.PrimitiveArray](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.PrimitiveArray)  :arrow_heading_up: 4 | 35
- [ ] [hpack](https://hydra.nixos.org/eval/1784125?filter=hpack)  :arrow_heading_up: 4 | 14
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196410349) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196422628) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196404708) [toplevel](https://hydra.nixos.org/eval/1784125?filter=hpack)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196413270) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196422105) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196420330) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc8107.hpack)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196412578) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196421947) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196413020) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc884.hpack)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196421895) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196415491) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196408410) [haskell.packages.ghc902](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc902.hpack)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196417023) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196413192) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196404905) [haskell.packages.ghc924](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc924.hpack)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196418155) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196423985) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196408177) [haskell.packages.ghc942](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc942.hpack)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196422379) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196419683) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196404998) [haskellPackages](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hpack)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196418651) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196416044) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196406499) [haskellPackages.BiobaseTypes](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.BiobaseTypes)  :arrow_heading_up: 3 | 21
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196420190) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196416211) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196418220) [haskellPackages.hw-json-standard-cursor](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hw-json-standard-cursor)  :arrow_heading_up: 2 | 6
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196424186) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196414471) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196417378) [haskellPackages.hw-json-simple-cursor](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hw-json-simple-cursor)  :arrow_heading_up: 2 | 4
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196418713) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196420289) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196413517) [haskellPackages.BiobaseENA](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.BiobaseENA)  :arrow_heading_up: 1 | 18
- [ ] [hoogle](https://hydra.nixos.org/eval/1784125?filter=hoogle)  :arrow_heading_up: 1 | 3
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196422115) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196405881) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196423603) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc8107.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196422786) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196414447) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196422971) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc884.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196418055) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196420840) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196407018) [haskell.packages.ghc902](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc902.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196410717) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196404547) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196416037) [haskell.packages.ghc924](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc924.hoogle)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196417447) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196420767) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196412797) [haskell.packages.ghc942](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc942.hoogle)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196415131) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196408119) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196421788) [haskellPackages](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hoogle)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196404383) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196414501) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196419241) [haskellPackages.hw-json](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hw-json)  :arrow_heading_up: 1 | 3
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196417524) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196410882) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196417535) [haskellPackages.bcp47](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.bcp47)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196407642) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196406100) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196405639) [haskellPackages.http3](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.http3)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196424401) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196420045) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196413468) [haskellPackages.persistent-postgresql-streaming](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.persistent-postgresql-streaming)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196406706) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196418453) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196415598) [haskellPackages.BiobaseXNA](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.BiobaseXNA)  :arrow_heading_up: 0 | 17
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196413899) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196421376) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196419235) [haskellPackages.BiobaseFasta](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.BiobaseFasta)  :arrow_heading_up: 0 | 3
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196415915) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196411921) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196405221) [haskellPackages.hw-dsv](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hw-dsv)  :arrow_heading_up: 0 | 3
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196424359) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196409394) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196421150) [haskellPackages.hw-json-lens](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hw-json-lens)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196415144) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196422804) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196421232) [haskellPackages.snaplet-persistent](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.snaplet-persistent)  :arrow_heading_up: 0 | 1
- [ ] [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196421393) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196420936) [haskellPackages.verifiable-expressions](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.verifiable-expressions)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196406130) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196421167) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196414487) [haskellPackages.align-audio](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.align-audio) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196410568) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196414551) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196405878) [haskellPackages.bcp47-orphans](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.bcp47-orphans) 
- [ ] [cabal2nix-unstable](https://hydra.nixos.org/eval/1784125?filter=cabal2nix-unstable) 
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196589928) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196589920) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196589904) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc8107.cabal2nix-unstable)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196589942) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196589940) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196589905) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc884.cabal2nix-unstable)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196589893) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196589968) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196589936) [haskell.packages.ghc902](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc902.cabal2nix-unstable)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196589953) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196589963) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196589914) [haskell.packages.ghc924](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc924.cabal2nix-unstable)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196589970) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196589898) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196589949) [haskell.packages.ghc942](https://hydra.nixos.org/eval/1784125?filter=haskell.packages.ghc942.cabal2nix-unstable)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196589957) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196589930) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196589902) [haskellPackages](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.cabal2nix-unstable)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196406227) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196421655) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196406293) [haskellPackages.easytensor-vulkan](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.easytensor-vulkan) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196409286) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196404435) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196411930) [haskellPackages.espial](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.espial) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196414835) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196421081) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196416236) [haskellPackages.esqueleto-streaming](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.esqueleto-streaming) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196418284) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196408785) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196408260) [haskellPackages.experimenter](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.experimenter) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196422142) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196424131) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196416860) [haskellPackages.harfbuzz-pure](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.harfbuzz-pure) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196422478) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196405678) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196407874) [haskellPackages.heroku-persistent](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.heroku-persistent) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196411034) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196419255) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196423954) [haskellPackages.hmatrix-nlopt](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hmatrix-nlopt) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196424245) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196423305) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196423322) [haskellPackages.hoauth2-providers-tutorial](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hoauth2-providers-tutorial) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196421891) [[:apple::hourglass::no_entry_sign:]](https://hydra.nixos.org/build/196411243) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196414901) [haskellPackages.hs-swisstable-hashtables-class](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hs-swisstable-hashtables-class) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196414958) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196418504) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196405818) [haskellPackages.hw-simd-cli](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.hw-simd-cli) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196408744) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196407451) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196423883) [haskellPackages.mywork](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.mywork) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196411647) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196408732) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196404618) [haskellPackages.om-kubernetes](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.om-kubernetes) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196417052) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196415951) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196418626) [haskellPackages.persistent-mtl](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.persistent-mtl) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196413168) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196420733) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196420924) [haskellPackages.persistent-pagination](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.persistent-pagination) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196416861) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196406824) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196415592) [haskellPackages.persistent-typed-db](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.persistent-typed-db) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196406939) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196422992) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196422899) [haskellPackages.rounded](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.rounded) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196408895) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196405809) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196411612) [haskellPackages.rounded-hw](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.rounded-hw) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196422097) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196424474) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196413853) [haskellPackages.sound-collage](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.sound-collage) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196423771) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/196420975) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196420852) [haskellPackages.unicode-names](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.unicode-names) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196405261) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196419877) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196406775) [haskellPackages.warp-quic](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.warp-quic) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/196417124) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/196419615) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/196410102) [haskellPackages.xbattbar](https://hydra.nixos.org/eval/1784125?filter=haskellPackages.xbattbar) 
</details>

#### Top 50 broken packages, sorted by number of reverse dependencies
<details><summary>50 job(s) </summary>

[amazonka-core](https://packdeps.haskellers.com/reverse/amazonka-core) :arrow_heading_up: 185  
[gogol-core](https://packdeps.haskellers.com/reverse/gogol-core) :arrow_heading_up: 184  
[haskell98](https://packdeps.haskellers.com/reverse/haskell98) :arrow_heading_up: 153  
[enumerator](https://packdeps.haskellers.com/reverse/enumerator) :arrow_heading_up: 56  
[util](https://packdeps.haskellers.com/reverse/util) :arrow_heading_up: 49  
[derive](https://packdeps.haskellers.com/reverse/derive) :arrow_heading_up: 48  
[amazonka](https://packdeps.haskellers.com/reverse/amazonka) :arrow_heading_up: 43  
[accelerate](https://packdeps.haskellers.com/reverse/accelerate) :arrow_heading_up: 42  
[parseargs](https://packdeps.haskellers.com/reverse/parseargs) :arrow_heading_up: 42  
[MonadCatchIO-transformers](https://packdeps.haskellers.com/reverse/MonadCatchIO-transformers) :arrow_heading_up: 41  
[data-lens](https://packdeps.haskellers.com/reverse/data-lens) :arrow_heading_up: 33  
[rank1dynamic](https://packdeps.haskellers.com/reverse/rank1dynamic) :arrow_heading_up: 33  
[distributed-static](https://packdeps.haskellers.com/reverse/distributed-static) :arrow_heading_up: 31  
[language-ecmascript](https://packdeps.haskellers.com/reverse/language-ecmascript) :arrow_heading_up: 31  
[distributed-process](https://packdeps.haskellers.com/reverse/distributed-process) :arrow_heading_up: 30  
[iteratee](https://packdeps.haskellers.com/reverse/iteratee) :arrow_heading_up: 29  
[jmacro](https://packdeps.haskellers.com/reverse/jmacro) :arrow_heading_up: 29  
[mmsyn3](https://packdeps.haskellers.com/reverse/mmsyn3) :arrow_heading_up: 28  
[autodocodec-yaml](https://packdeps.haskellers.com/reverse/autodocodec-yaml) :arrow_heading_up: 27  
[crypto-numbers](https://packdeps.haskellers.com/reverse/crypto-numbers) :arrow_heading_up: 25  
[either-unwrap](https://packdeps.haskellers.com/reverse/either-unwrap) :arrow_heading_up: 25  
[sydtest](https://packdeps.haskellers.com/reverse/sydtest) :arrow_heading_up: 24  
[crypto-pubkey](https://packdeps.haskellers.com/reverse/crypto-pubkey) :arrow_heading_up: 22  
[haskelldb](https://packdeps.haskellers.com/reverse/haskelldb) :arrow_heading_up: 22  
[wxdirect](https://packdeps.haskellers.com/reverse/wxdirect) :arrow_heading_up: 22  
[alg](https://packdeps.haskellers.com/reverse/alg) :arrow_heading_up: 21  
[amazonka-s3](https://packdeps.haskellers.com/reverse/amazonka-s3) :arrow_heading_up: 21  
[mmsyn2](https://packdeps.haskellers.com/reverse/mmsyn2) :arrow_heading_up: 21  
[wxc](https://packdeps.haskellers.com/reverse/wxc) :arrow_heading_up: 21  
[biocore](https://packdeps.haskellers.com/reverse/biocore) :arrow_heading_up: 20  
[wxcore](https://packdeps.haskellers.com/reverse/wxcore) :arrow_heading_up: 20  
[attoparsec-enumerator](https://packdeps.haskellers.com/reverse/attoparsec-enumerator) :arrow_heading_up: 19  
[bytestring-show](https://packdeps.haskellers.com/reverse/bytestring-show) :arrow_heading_up: 19  
[fay](https://packdeps.haskellers.com/reverse/fay) :arrow_heading_up: 19  
[wx](https://packdeps.haskellers.com/reverse/wx) :arrow_heading_up: 19  
[asn1-data](https://packdeps.haskellers.com/reverse/asn1-data) :arrow_heading_up: 18  
[dbus-core](https://packdeps.haskellers.com/reverse/dbus-core) :arrow_heading_up: 18  
[gtksourceview2](https://packdeps.haskellers.com/reverse/gtksourceview2) :arrow_heading_up: 18  
[ukrainian-phonetics-basic](https://packdeps.haskellers.com/reverse/ukrainian-phonetics-basic) :arrow_heading_up: 18  
[HGamer3D-Data](https://packdeps.haskellers.com/reverse/HGamer3D-Data) :arrow_heading_up: 17  
[certificate](https://packdeps.haskellers.com/reverse/certificate) :arrow_heading_up: 17  
[dbus-client](https://packdeps.haskellers.com/reverse/dbus-client) :arrow_heading_up: 17  
[gconf](https://packdeps.haskellers.com/reverse/gconf) :arrow_heading_up: 17  
[gtk-serialized-event](https://packdeps.haskellers.com/reverse/gtk-serialized-event) :arrow_heading_up: 17  
[cuda](https://packdeps.haskellers.com/reverse/cuda) :arrow_heading_up: 16  
[happstack-jmacro](https://packdeps.haskellers.com/reverse/happstack-jmacro) :arrow_heading_up: 16  
[manatee-core](https://packdeps.haskellers.com/reverse/manatee-core) :arrow_heading_up: 16  
[monads-fd](https://packdeps.haskellers.com/reverse/monads-fd) :arrow_heading_up: 16  
[tls-extra](https://packdeps.haskellers.com/reverse/tls-extra) :arrow_heading_up: 16  
[ADPfusion](https://packdeps.haskellers.com/reverse/ADPfusion) :arrow_heading_up: 15  
</details>


*:arrow_heading_up:: The number of packages that depend (directly or indirectly) on this package (if any). If two numbers are shown the first (lower) number considers only packages which currently have enabled hydra jobs, i.e. are not marked broken. The second (higher) number considers all packages.*

*Report generated with [maintainers/scripts/haskell/hydra-report.hs](https://github.com/NixOS/nixpkgs/blob/haskell-updates/maintainers/scripts/haskell/hydra-report.sh)*

sternenseemann committed:
Merge remote-tracking branch 'origin/master' into haskell-updates

sternenseemann committed:
taffybar: accurately report version

sternenseemann commented:
I'm tentatively attributing the `rel8` failure to rosetta or the weirdness with darwin builders atm – we'll just see.

sternenseemann commented:
@ofborg build haskellPackages.rel8

sternenseemann committed:
ihaskell: temporarily remove ihaskell-diagrams from the environment

ihaskell-display was changed recently to no longer use the
switchToTmpDir IO action. This change also affected ihaskell-diagrams,
but only a new version of ihaskell-display has been released.
ihakell-diagrams' latest hackage version thus depends on a removed IO
action from ihaskell-display and can't be compiled.

sternenseemann committed:
ihaskell: test on Hydra, as mistune is no longer marked insecure

sternenseemann committed:
haskellPackages: mark builds failing on hydra as broken

This commit has been generated by maintainers/scripts/haskell/mark-broken.sh
